src/python/poetry.ts

```diff
@@ -366,6 +366,13 @@ export interface PoetryPyprojectOptionsWithoutDeps {
    * E.g. "Bug Tracker"
    */
   readonly urls?: { [key: string]: string };
+  /**
+   * Package mode (optional).
+   * @see https://python-poetry.org/docs/pyproject/#package-mode
+   * @default true
+   * @example false
+   */
+  readonly "package-mode"?: boolean;
```

Comment:
This should already automatically be converted into kebab case.

```suggestion
  readonly packageMode?: boolean;
```

Reply:
I tried that first and I believe it ended up snake cased

Reply:
Looks like we need to change the separator on `decamelizeKeysRecursively` since poetry uses kebab `-` and not snake `_`